scss/tech.scss

```diff
@@ -0,0 +1,42 @@
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: column wrap;
+  margin: 15px;
+  padding: 10px;
+  position: relative;
+  text-align: center;
+  width: 190px;
+
+  &::before {
+    border-radius: 5px;
+    border: 1px solid $green;
+    content: '';
+    height: 241px;
+    margin: 10px;
+    position: absolute;
+    right: 0;
```

Comment:
the idea of this border is to have the same size as the child but moved to the top and the left.

I'd say this should be changed to `left: -20px;`